docs/overview/public_vs_permissioned.md

```diff
@@ -0,0 +1,115 @@
+- How value is bridged in-to / out-of the ecosystem
+- How the validator set of the chain is established - most common is Proof of Stake (PoS)
+- How data availability is maintained - to check the working of the validators ensure the historical state is not lost
+- The consensus algorithm, and how it interacts with the consensus of other blockchains
+- How state in a Layer 2 is provable in a parent Layer 1 chain (rollup technologies etc.)
+
+## Common public considerations
+
+The thing most consistent across public blockchain technologies, is that the technical decision are
+backed by a system of token economics.
+
+Put simply, creating a system where it's more financially rewarding to behave honestly, than it
+is to subvert and cheat the system.
+
+This means that participation costs, and that the mechanisms needed to reliably get your transactions
+into these systems are complex. Also that the time it might take to get a transaction onto the chain
+can be much longer than for a permissioned blockchain, with the potential to have to make a number
+of adjustments/resubmissions.
+
+The choice of whether to run your own node, or use a managed API, to access these blockchain ecosystems
+is also a factor in the behavior of the transaction submission and event streaming.
+
+## FireFly architecture for public chains
+
+One of the fastest evolving aspects of the Hyperledger FireFly ecosystem, is how it facilitates
+enterprises to participate in these.
+
+[![FireFly Public Transaction Architecture](../images/firefly_transaction_manager.jpg)](https://github.com/hyperledger/firefly-transaction-manager)
+
+The architecture is summarized as follows:
+
+- New **FireFly Transaction Manager** runtime
+  - Operates as a microservice extension of the FireFly Core
+  - Uses the `operation` resource within FireFly Core to store and update state
+  - Runs as a singleton and is responsible for `nonce` assignment
+  - Takes as much heavy lifting away from blockchain specific connectors as possible
+- Lightweight FireFly Connector API (`ffcapi`)
+  - Simple synchronous RPC operations that map to the most common operations supported across public blockchain technologies
+  - Examples:
+    - Find the next nonce for a given signing key
+    - Serialize a transaction from a set of JSON inputs and an interface definition
+    - Submit an un-signed transaction with a given gas price to the blockchain, via a signing wallet
+    - Establish a new block listener
+    - Poll for new blocks
+    - Establish a new event log listener
+    - Poll for new events
+- Pluggable **Policy Engine**
+  - Invoked to make decisions on transaction submission
+  - Responsible for gas price calculation
+  - Able to intervene and adjust the characteristics of signing/submission
+  - OSS reference implementation provided with Gas Station REST API integration
+- **Confirmation Manager**
+  - Extracted from the EthConnect codebase
```

Comment:
Is it Ethconnect or EthConnect? The header in that repo says Ethconnect, so that's how I've always typed it.

Reply:
I'll punt for `Ethconnect` (which is certainly how I first spelt it), but it seems to be leaning towards `EthConnect` ... since `FabConnect` came along 🤷